utils/db.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+var client *mongo.Client
+
+func GetDB() *mongo.Database {
+	return ResolveClientDB().Database(DbName)
+}
+
+func ResolveClientDB() *mongo.Client {
+	if client != nil {
+		return client
+	}
+
+	var err error
+	clientOptions := options.Client().ApplyURI(MongoDB)
+	client, err = mongo.Connect(context.Background(), clientOptions)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// check the connection
+	err = client.Ping(context.Background(), nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return client
+}
+
+func CloseClientDB() {
```

Comment:
I noticed that you don't call the `CloseClientDB` method anywhere in your implementation. Is there any specific reason for that?